POLICY.txt

```diff
@@ -34,6 +37,8 @@ it reports one of the following problems:
 - Publicly accessible version strings of used software.
 - Security vulnerablities that can only be used within the scope of the
   used account.
+- The vulnerability exists in a third party software and is already
```

Comment:
That would exclude about everything. What about something along the lines of:
- The vulnerability is known and has a CVSS score less than 7
- Deviations from best practices or common configuration that do not have a direct security impact